src/namecoin.py

```diff
@@ -235,9 +234,9 @@ def queryHTTP(self, data):
                 result = resp.read()
                 if resp.status != 200:
                     raise Exception("Namecoin returned status %i: %s" % (resp.status, resp.reason))
-            except:
+            except:  # noqa:E722
                 logger.info("HTTP receive error")
-        except:
+        except:  # noqa:E722
```

Comment:
move this `except` up to after `self.con.send(data)` and put the second try/except block outside of the first one.

Reply:
Updated